src/routes.ts

```diff
@@ -0,0 +1,39 @@
+import { Router } from "express";
+import { RpcBundle, JsonRpcRequest } from "./models";
+import { S3Uploader } from "./upload";
+import config from "./config";
+import log from "./log";
+
+const routes = Router();
+const upload = new S3Uploader(config);
+
+routes.get("/", (req, res) => {
+  return res.json({ message: "Hello World" });
+});
+
+routes.post("/", async (req, res) => {
+  try {
+    const request: JsonRpcRequest = req.body;
+    log.trace(`Handling incoming request: ${JSON.stringify(request)}`);
+    if (request.method != "eth_sendBundle") {
+      throw "unsupported method";
+    }
+    if (request.params.length != 1) {
+      throw "expecting a single bundle";
+    }
+    const bundle: RpcBundle = request.params[0];
+    log.debug(`Received Bundle: ${JSON.stringify(bundle)}`);
+
+    await upload.upload(bundle, request.id);
```

Comment:
Nit: if we called the constant "aws" or "s3bucket" (which seems reasonable since its essentially an instance of an object object requiring AWS credentials), then this would read as `aws.upload`...